frontend/src/pages/workspace-list/ui/workspace-list-page.module.css

```
@@ -1,31 +1,94 @@
+    #ffffff;
+  box-shadow: 0 20px 48px rgba(0, 0, 0, 0.04);
+}
+
+.heroCopy {
+  max-width: 760px;
+  display: grid;
+  gap: 12px;
+}
+
+.eyebrow {
+  font-family: "Geist Mono", "SFMono-Regular", monospace;
```

Stylelint (17.9.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)